setup.py

```diff
@@ -854,22 +854,24 @@ def detect_modules(self):
                                depends=['socketmodule.h'],
                                libraries=math_libs) )
         # Detect SSL support for the socket module (via _ssl)
-        search_for_ssl_incs_in = [
-                              '/usr/local/ssl/include',
-                              '/usr/contrib/ssl/include/'
-                             ]
-        ssl_incs = find_file('openssl/ssl.h', inc_dirs,
-                             search_for_ssl_incs_in
-                             )
+        if 'AS_DEPENDENCIES_DIR' in os.environ:
```

Comment:
TO what am I supposed to set `AS_DEPENDENCIES_DIR` when compiling myself?

Reply:
In our build the `AS_DEPENDENCIES_DIR` sits under `usr` so whatever is the equivalent.  This is the path to our `libssl.so`

`AS_DEPENDENCIES_DIR/usr/lib/libssl.so`